src/modules/launcher/Plugins/Community.PowerToys.Run.Plugin.WebSearch/ExtraStringMethods.cs

```diff
@@ -0,0 +1,82 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Community.PowerToys.Run.Plugin.WebSearch
+{
+    public static class ExtraStringMethods
```

Comment:
Why are we doing this?

Reply:
because we need these extra methods that work with strings?